variables.tf

```diff
@@ -335,6 +335,12 @@ variable "cluster_config_endpoint_type" {
   }
 }
 
+variable "enable_ocp_console" {
+  description = "Flag to specify whether to enable or disable the OpenShift console patch."
```

Comment:
Remove the word "patch" from the description

Reply:
Done